Tests/Form/Type/BooleanTypeTest.php

```diff
@@ -85,11 +85,11 @@ public function testOptions()
         ), $resolvedOptions);
     }
 
-    public function testDeprecatedCatalogueOption()
+    public function testLegacyCatalogueOption()
```

Comment:
I renamed the function with ``Legacy`` to respect the ``symfony/phpunit-bridge`` convention.
This function tests the legacy option ``catalogue``, using ``configureOptions`` is not related to the name of the function.

And what do you mean by "both tests" ?